arches_resource_sets/views/resource_set_members_bulk_view.py

```diff
@@ -0,0 +1,24 @@
+from arches.app.utils.betterJSONSerializer import JSONDeserializer
+from arches.app.utils.response import JSONErrorResponse, JSONResponse
+from arches.app.views.api import APIBase
+from arches_resource_sets.models import ResourceSet
+
+
+class ResourceSetMembersBulkView(APIBase):
+    def post(self, request, set_id):
+        request_body = JSONDeserializer().deserialize(request.body)
+        resource_ids = request_body["resource_ids"] if "resource_ids" in request_body else ""
+        operation = request_body["operation"] if "operation" in request_body else "add"
+
+        try:
+            set = ResourceSet.objects.get(id=set_id)
+
+        except ResourceSet.DoesNotExist:
```

Comment:
put deserialization in the try/catch block. Make sure to catch that error too.

```suggestion
        try:
            request_body = JSONDeserializer().deserialize(request.body)
            resource_ids = request_body["resource_ids"] if "resource_ids" in request_body else ""
            operation = request_body["operation"] if "operation" in request_body else "add"
            set = ResourceSet.objects.get(id=set_id)
        except ResourceSet.DoesNotExist:
```